applicationinsights.json

```diff
@@ -8,6 +8,9 @@
   "instrumentation": {
     "logging": {
       "level": "DEBUG"
+    },
+    "redis": {
+      "enabled": false
```

Comment:
i noticed the guidance suggests sampling redis at 5% which i _could_ see being useful just as a sanity check that we are using caching. BUT - i don't recall ever making use of these redis logs in the last year so i think it's fine to completely disable 👍

Reply:
Yeah that was my thinking - chatting to other folks no one else seems to use it either. If it is important we can just turn it on again